psi4/driver/procrouting/response/scf_response.py

```diff
@@ -681,4 +685,101 @@ def tda_solver(e, n, g, m):
 
     core.print_out("\n")
 
+# Print out TDM vectors just length representation for now: other forms could be added)
```

Comment:
The transition moments are not observable: I'd prefer them to be printed only with a high print level. In the output file they're mostly useful only for debugging, I believe.

Reply:
We use TDM vectors extensively for modelling the alignment of dye molecules within liquid crystal guest-host systems, i.e. how well is a dye transition likely to be aligned with the bulk of the liquid crystal phase.